src/components/App/MapLayout.tsx

```diff
@@ -50,34 +52,36 @@ export default function MapLayout({
 }) {
   const app = React.useContext(AppContext)
   const { clientSideConfiguration } = app || {}
-  const [isMenuOpen, setIsMenuOpen] = React.useState(false)
   const firstStart = isFirstStart()
-  const toggleMainMenu = React.useCallback((newValue?: boolean) => {
-    setIsMenuOpen(typeof newValue === 'boolean' ? newValue : !isMenuOpen)
-  }, [isMenuOpen])
+  const router = useRouter()
+  const isOnboardingVisible = firstStart || router.pathname === '/onboarding'
```

Comment:
This is a part of the fix to get the missing onboarding dialog back when clicking the logo in the upper left.